integration/windows_hostprocess_test.go

```diff
@@ -127,12 +127,14 @@ func runHostProcess(t *testing.T, expectErr bool, image string, action hpcAction
 	action(t, cn, containerConfig)
 }
 
-func startAndTestContainer(t *testing.T, sb string, sbConfig *runtime.PodSandboxConfig, cnConfig *runtime.ContainerConfig) {
+func createStartStopAndRemoveTestContainer(t *testing.T, sb string, sbConfig *runtime.PodSandboxConfig, cnConfig *runtime.ContainerConfig) {
 	t.Log("Create the container")
 	cn, err := runtimeService.CreateContainer(sb, cnConfig, sbConfig)
 	require.NoError(t, err)
 	t.Log("Start the container")
 	require.NoError(t, runtimeService.StartContainer(cn))
+	// Wait few seconds for the container to be completely initialized
+	time.Sleep(5 * time.Second)
```

Comment:
Hmm, isn't CreateContainer synchronous? What does it return if we call StopContainer too early?

Reply:
(I think its more about how the underlying compute system deals with it? )
its scheduling dependent - Its possible to hit a window where the container is not running yet but the kill has been sent immediately

Reply:
Yea I'd expect this too.. I'd think if start returns we should expect the workload to be running. I'd imagine anything thats not that case is a bug (if the command you asked to run is expected to stay alive and isn't "echo foo" or some short lived task obviously).

Reply:
@mikebrow , Not sure if the window between container start and container stop is really needed. Thoughts?

Reply:
I've not looked around for the cplatpublic.azurecr.io/args-escaped-test-image-ns code that's being tested..

Reply:
Looked over the runtimeService.StartContainer() code again and I see that if it returns, the container has started successfully.  If any windows exists where the container is not running, I think the 5 sec delay is more than sufficient in this case as the test image is only looking for how CMD and ENTRYPOINTs are overriden and if they start up as expected